liblll/Compiler.cpp

```diff
@@ -28,13 +28,16 @@ using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
-bytes dev::eth::compileLLL(string const& _src, bool _opt, vector<string>* _errors)
+
+bytes dev::eth::compileLLL(string const& _src, bool _opt, vector<string>* _errors, ReadCallback const& _readFile)
 {
+	CodeFragment::ReadCallback readFile = [&](string const& _name) { return _readFile(_name); };
```

Comment:
I don't think you need this wrapper. Can't you pass `_readFile` as the callback?

Reply:
I guess it should work.